core/src/main/scala/com/salesforce/op/stages/impl/feature/HumanNameDetector.scala

```diff
@@ -89,7 +91,7 @@ class HumanNameDetectorModel[T <: Text]
   val treatAsName: Boolean,
   val orderedGenderDetectStrategies: Seq[GenderDetectStrategy] = Seq.empty[GenderDetectStrategy]
 )(implicit tti: TypeTag[T])
-  extends UnaryModel[T, NameStats]("human name detector", uid) with NameDetectFun {
+  extends UnaryModel[T, NameStats]("humanNameDetect", uid) with NameDetectFun {
```

Comment:
make `operationName` a ctor argument for the class so it would be easier to debug